test/base.test.js

```diff
@@ -461,3 +461,41 @@ test('should handle custom signer', async (t) => {
   t.equal(response.headers['set-cookie'].includes(DEFAULT_ENCRYPTED_SESSION_ID), false)
   t.match(response.headers['set-cookie'], /sessionId=[\w-]{32}.[\w-%]{43,135}; Path=\/; HttpOnly; Secure/)
 })
+
+test('should set session cookie if saveUninitialized is false and maxAge is on', async (t) => {
```

Comment:
This unit test is contradicting line 383 test: "should not set session cookie if no data in session and saveUninitialized is false"

Reply:
There's a key difference in these tests though! In this test, the session is already initialized. 

`saveUninitialized: false` should only prevent uninitialized sessions from being saved. Existing sessions should still be saved